Description: Apply #!/usr/bin/perl to all installed Perl scripts
Author: Colin Watson <cjwatson@debian.org>
Forwarded: no
Last-Update: 2010-05-26

Index: b/decrypt/decr
===================================================================
--- a/decrypt/decr
+++ b/decrypt/decr
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # This script will decrypt a Perl script that has been encrypted using the
 # "encrypt" script. It cannot decrypt any other kind of encrypted Perl script.
Index: b/decrypt/encrypt
===================================================================
--- a/decrypt/encrypt
+++ b/decrypt/encrypt
@@ -1,3 +1,4 @@
+#!/usr/bin/perl
 
 require 5.002 ;
 
Index: b/examples/filtdef
===================================================================
--- a/examples/filtdef
+++ b/examples/filtdef
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use strict ;
 use warnings ;
Index: b/examples/filtuu
===================================================================
--- a/examples/filtuu
+++ b/examples/filtuu
@@ -1,3 +1,4 @@
+#!/usr/bin/perl
 
 print "use Filter::UUdecode ;\n" ;
 while (<>) {
